templates/io-utils-search-replace-form.html.twig

```diff
@@ -0,0 +1,22 @@
+{#
+/**
+ * @file
+ * Theme implementation for the Search and Replace form.
+ */
+#}
+
+<div class="search-replace-form">
+  {{ form.help_link }}
```

Comment:
Instead of calling each form component individually, standard forms like this are usually better left to the auto-renderer. If you have a specific use case like this, you may need to add a comment to let other developers know why.